spec/forms/schools/add_participants/wizard_steps/confirm_mentor_transfer_step_spec.rb

```diff
@@ -20,17 +20,26 @@
         step.transfer_confirmed = "yes"
       end
 
-      context "when a training choice has been made" do
+      context "when no training choice has been made" do
+        it "returns :need_training_setup" do
+          allow(wizard).to receive(:destination_school_cohort).and_return(nil)
+          expect(step.next_step).to eql :need_training_setup
+        end
+      end
+
+      context "when a FIP training choice has been made" do
         it "returns :none" do
-          allow(wizard).to receive(:need_training_setup?).and_return(false)
+          allow(wizard).to receive(:destination_school_cohort).and_return(true)
```

Comment:
shouldn't the `destination_school_cohort` return a `SchoolCohort` record or `nil`?